railties/lib/rails/generators/rails/app/templates/config/initializers/active_record_destroy_all_in_batches.rb

```diff
@@ -0,0 +1,5 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
I have put the config in an initializer, so we don't promote difference between environments. However, now we have both this file and `config/initializers/active_record_belongs_to_required_by_default.rb`. Should we consider collecting all of the Active Record defaults in one file or changing the `belongs_to_required_by_default` default value for 5.0?